### PR TITLE
[WIP] Added the option to set custom break tag

### PR DESCRIPTION
**Pull request by oleksandr-kuzmenko**
The default break element is `<br>`, but now you can set custom element.

[edit] ~fix #~﻿~25~

### CODE REVIEW

**Comment by oleksandr-kuzmenko** [comment]
@sstur what do you think about this PR?

File: packages/draft-js-export-html/README.md [hunk 1]
```diff
@@ -141,6 +141,23 @@ let options = {
...
+Example:
+
+```js
+const contentState = convertFromRaw(
+  { entityMap: {}, blocks: [{ text: '', ... }] },
+);
+
+stateToHTML(contentState);  // '<p><br></p>
+
+let options = { breakElement: '<hr />'} ;
```
Comment (sstur) [inline packages/draft-js-export-html/README.md]: Nit: space on the left, no space on the right.

File: packages/draft-js-export-html/src/__tests__/stateToHTML-test.js [hunk 2]
```diff
@@ -241,4 +241,32 @@ describe('stateToHTML', () => {
...
+    expect(stateToHTML(contentState)).toBe(
+      '<p><br></p>',
+    );
+
+    expect(stateToHTML(contentState, {breakElement: ''})).toBe(
+      '<p></p>',
+    );
+
+    expect(stateToHTML(contentState, {breakElement: '<hr />'})).toBe(
+      '<p><hr /></p>',
```
Comment (sstur) [inline packages/draft-js-export-html/src/__tests__/stateToHTML-test.js]: Let's use `<hr>` with the closing `/`

**Review by sstur — CHANGES_REQUESTED** [review]
Thanks for this contribution.

Your code is good. I am OK with the option of overriding the default `BREAK`.

Few thoughts tho:
1. I don't think we should call it `breakElement` maybe just `linebreak` since it is not necessarily an "element".
2. I think we should separate the two different uses of `<br>`
   - one is for preventing collapse of an empty block element `<p><br></p>`
   - the other is for replacing `\n` in text nodes.
  They should each have their own option.

Side note, this doesn't exactly close #25 in my opinion. I think the best solution for that issue is: "add option to not render lines (blocks) which have only whitespaces?"

Regardless of #25 tho, I like your PR and I think we should split this into two options and then land it.

**Comment by oleksandr-kuzmenko** [comment]
> 2. I think we should separate the two different uses of <br>

Make sense. I agree.

@sstur, how do you want to call one more option (for replacing `\n` in text nodes)?

**Comment by MihaiWill** [comment]
What happened? I can't change it anymore. All my line breaks are with '\n' instead of <br/>. Please help!

**Pull request closed without merging**